flang/test/lit.cfg.py

```diff
@@ -122,6 +122,7 @@
 # the build directory holding that tool.
 tools = [
     ToolSubst("%flang", command=FindTool("flang-new"), unresolved="fatal"),
+    ToolSubst("%clang", command=FindTool("clang"), unresolved="fatal"),
```

Comment:
@banach-space I had to add this, as I need both flang and clang to compile the test.  This could move to a different PR if needed.  Please advise!

Reply:
Remove this if using the system compiler works.

Reply:
I have removed this in favor of compiling a LL file instead, like suggest below.